packages/nextjs/app/debug/_components/contract/Array.tsx

```diff
@@ -74,16 +74,15 @@ export const ArrayInput = ({
                     nextInputObject = nextInputRecipe(parentForm!);
```

Comment:
Could we assign `setForm` property with:
```
(
  nextInputRecipe: Record<string, any> | ((arg: Record<string, any>) => Record<string, any>),
) => {
  let nextInputObject: Record<string, any> = typeof nextInputRecipe === "function"
    ? nextInputRecipe(parentForm!)
    : nextInputRecipe;

  setInputArr((currentInputArray: any) => ({
    ...currentInputArray,
    [index]: nextInputObject[index] || null,
  }));
}
```
is it a bit more readable?

Reply:
thanks for the review, i think sometimes wrapping stuff in ternary operators do not necessarily mean they are readable. Can make some changes and update the comments